product_docs/docs/pgd/5/cli/command_ref/pgd_show-nodes.mdx

```diff
@@ -1,101 +1,61 @@
 ---
-title: show-nodes
+title: set-node-options
```

Comment:
This is the content of the show-nodes docs page, but it contains set-node-options. This suggests that something has gone awry in the importing.

Reply:
Yes, noticed that. Was waiting for few other changes also. Will push the change.